client/src/components/List/List.js

```diff
@@ -81,3 +32,7 @@ const ListContainer = styled.div`
     max-width: 50rem;
   }
 `;
+
+List.propTypes = {
+  items: PropTypes.array,
+};
```

Comment:
I think the proptypes should be declared at the beginning of the code.

Reply:
@jennydely this is not important. It's more common to define them at the end of the files.
See https://reactjs.org/docs/typechecking-with-proptypes.html or popular libs like material-ui:
https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/Button/Button.js